ArcGISRuntimeSDKQt_QMLSamples/Layers/CreateAndSaveKmlFile/CreateAndSaveKmlFile.qml

```diff
@@ -0,0 +1,194 @@
+// [WriteFile Name=CreateAndSaveKmlFile, Category=Layers]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+import QtQuick.Dialogs 1.2
+import Esri.ArcGISRuntime 100.6
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    property bool styling: false
+    property var kmlLayer
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Map {
+            BasemapTopographic {}
+        }
+
+        Button {
+            anchors{
+                left: parent.left
+                top: parent.top
+                margins: 3
+            }
+            text: qsTr("Save kmz file")
+
+            onClicked: {
+                fileDialog.visible = true;
+            }
+        }
+
+        GraphicsOverlay {
+            id: graphicsOverlay
+        }
+    }
+
+    KmlDocument {
+        id: kmlDocument
+        name: qsTr("KML Sample Document")
+        onSaveStatusChanged: {
+            if (saveStatus === Enums.TaskStatusErrored) {
+                console.log("Error: %1 - %2".arg(error.message).arg(error.additionalMessage))
+            }
+        }
+    }
+
+    Graphic {
+        id: polygonGraphic
+    }
+
+    Graphic {
+        id: polylineGraphic
+    }
+
+    Graphic {
+        id: pointGraphic
+    }
+
+    SimpleFillSymbol {
+        id: polygonSymbol
+        style: Enums.SimpleFillSymbolStyleSolid
+        color: "red"
+    }
+
+    SimpleLineSymbol {
+        id: polylineSymbol
+        style: Enums.SimpleLineSymbolStyleSolid
+        color: "blue"
+        width: 5
+        antiAlias: true
+    }
+
+    SimpleMarkerSymbol {
+        id: pointSymbol
+        style: Enums.SimpleMarkerSymbolStyleDiamond
+        color: "green"
+        size: 10
+    }
+
+    PolygonBuilder {
+        id: polygonBuilder
+        spatialReference: SpatialReference.createWgs84();
```

Comment:
Would be preferable when using declarative syntax to just declare the object out instead of using the function, just as a best practice (generally not good practice to bind properties to functions unless unavoidable):

```
PolygonBuilder {
    id: polygonBuilder
    spatialReference: SpatialReference { wkid: 4326 }
}
```